<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-C0TGEICA" ignored="0" name="Surefire test" tests="19" failures="4" timestamp="2023-01-11T20:58:04 EET" time="202.459" errors="0">
  <testcase name="runScenario" time="14.626" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="9.198" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="9.695" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="8.162" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="9.434" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	The resulted product no. 1 does not contain the searched keyword: book expected [true] but found [false],
	The resulted product no. 5 does not contain the searched keyword: book expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
The resulted product no. 1 does not contain the searched keyword: book expected [true] but found [false],
The resulted product no. 5 does not contain the searched keyword: book expected [true] but found [false]
at org.example.stepDefinitions.D04_searchSteps.searchSuccessful(D04_searchSteps.java:38)
at ✽.the relevant results of "book" are displayed(file:///D:/FET/EcommerceAutomation/src/main/resources/Features/F04_search.feature:8)
... Removed 2 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="8.913" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="8.061" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="10.143" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="8.853" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="8.91" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="10.48" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="8.456" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	The redirection URL is incorrect expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
The redirection URL is incorrect expected [true] but found [false]
at org.example.stepDefinitions.D06_homeSliderSteps.firstSliderRedirection(D06_homeSliderSteps.java:37)
at ✽.user is redirected to the page of the first product(file:///D:/FET/EcommerceAutomation/src/main/resources/Features/F06_homeSlider.feature:8)
... Removed 2 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="8.417" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	The redirection URL is incorrect expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
The redirection URL is incorrect expected [true] but found [false]
at org.example.stepDefinitions.D06_homeSliderSteps.secondSliderRedirection(D06_homeSliderSteps.java:47)
at ✽.user is redirected to the page of the second product(file:///D:/FET/EcommerceAutomation/src/main/resources/Features/F06_homeSlider.feature:13)
... Removed 2 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="15.212" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="15.805" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="10.261" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	The URL the user was directed to is incorrect expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
The URL the user was directed to is incorrect expected [true] but found [false]
at org.example.stepDefinitions.D07_followUsSteps.redirect(D07_followUsSteps.java:56)
at ✽."https://demo.nopcommerce.com/new-online-store-is-open" is opened in new tab(file:///D:/FET/EcommerceAutomation/src/main/resources/Features/F07_followUs.feature:14)
... Removed 2 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="15.975" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="8.653" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="12.325" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
</testsuite> <!-- Surefire test -->
